packages/devtools_shared/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 4.0.0
+
+- Adds a `DeeplinkApi.androidBuildVariants` endpoint to ServerAPi.
+- **BREAKING CHANGE**:
+  - ServerApi.handle extensionsManager, api parameter are converted to named parameters
```

Comment:
nit: add tics and reword a bit
"`ServerApi.handle` parameters `extensionsManager` and `api` were converted to be named parameters"